src/sections/Blocks.section.tsx

```diff
@@ -303,7 +303,13 @@ export const BlocksSection: React.FC<{ address?: string }> = ({ address }) => {
           currentPage,
         }}
         columns={columns}
-        contentEmpty={<h3>You haven't mined any blocks yet</h3>}
+        contentEmpty={
+          <h3>
+            {!!address
+              ? "You haven't mined any blocks yet"
+              : 'Unable to retrieve pool data'}
```

Comment:
For consistency, please use either ' or ".

Reply:
i will change to `, default formatting is `'` but because there is `'` in the text, prettier wrapped it with `"`

Reply:
Ah yes